Firestore/Example/Podfile

```diff
@@ -54,6 +55,8 @@ def configure_local_pods()
   # to its podspec can still function. See Firestore-*-xcodebuild in
   # scripts/install_prereqs.sh for more details.
   pod 'FirebaseCore', :path => '../..'
+  pod 'FirebaseCoreDiagnostics', :path => '../..'
```

Comment:
@wilhuff This change is needed until Firebase 6.23.0 is public and could be undone then

Reply:
SG